src/ui/pages/list/PageTreePageContainer.js

```diff
@@ -13,6 +13,7 @@ export const mapStateToProps = state => ({
 
 export const mapDispatchToProps = dispatch => ({
   onWillMount: () => {
+    dispatch(clearErrors());
```

Comment:
This will not show server errors from other pages (like Add page).
If you add a page with an already used code, you won't see the error. This problem has been recently fixed